innersource-portal.md

```diff
@@ -0,0 +1,48 @@
+
+## Forces
+
+* There are multiple engineering teams in your organization but few channels through which they can effectively partner on shared challenges.
+* It is difficult for individuals to discover what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no clearly defined channel through which to advertise their projects to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+
+## Context
+
+* Managers have given tacit acceptance that their employees should take part in InnerSource projects
+* Employees desire the ability to be made aware of all of the InnerSource project activity occurring across the compnany
+* Owners of InnerSource projects desire the ability to attract as many potential contributors to their projects as possible
+* InnerSource project owners desire full control over the information being communicated concerning their projects
+
+## Solutions
+
+Create an InnerSource Portal intranet website where InnerSource project owners can easily advertise the availability of their projects. Visitors to the InnerSource portal should be able to look across all available projects as well as search for specific projects based on various criteria such as project name, technologies in use, contributor names, sponsoring business unit etc.  The information displayed via the InnerSource portal should be under the full control of the InnerSource project owners at all times and they should have the ability to add or remove projects as needed.  Project owners should include all relevant information concerning their projects within the listings including the owner’s name, the project name and description as well as links to the code repository and any supporting documentation.
+
+## Resulting Context
+
+The InnerSource Portal has enabled InnerSource project owners to advertise their projects to an organization-wide audience.  Due to this increased visibility they are attracting much larger communities of contributors than ever before.  For those looking to get involved in InnerSource projects, the InnerSource portal has allowed them to discover exactly the kind of opportunities they are interested in by searching across all available InnerSource projects simultaneously based on their specific criteria.  Satisfying the needs of both of these audiences has helped establish InnerSource as a viable and attractive option for all areas of the organization to leverage to accomplish things together they could not have otherwise. 
+
+
+## Known instances
+
+* A large financial services organization has used the creation of an InnerSource Gig Marketplace website to foster their InnerSource program. 
```

Comment:
Is the InnerSource Gig Marketplace website == InnerSource Portal? Or is this mention just left over from the split?

Reply:
Yes.  This was an oversight on my part.  I'll remove the mention of the Gig Marketplace from this section of the pattern.